packages/@vivliostyle/theme-base/css/common/meta-properties.css

```
@@ -0,0 +1,56 @@
+:root {
+  /**
+   * Typography
+   */
+  --vs-font-family: sans-serif;
+  --vs-font-feature-settings: 'kern', 'liga', 'calt';
+  --vs-font-size: 16px;
+  --vs-font-size-smaller: calc(var(--vs-font-size) * 0.8);
+  --vs-font-style: normal;
+  --vs-font-weight-normal: 400;
+  --vs-font-weight-bold: 700;
+  --vs-hanging-punctuation: allow-end;
```

hanging-punctuation はデフォルトでは有効にするべきでないと思います。

句読点のぶら下げを使うためには、ぶら下げの領域を確保する必要があります。
テーブルセル内など、枠で囲んだボックス内のテキストなど、ぶら下げを考慮してpaddingを設定するなどの考慮が必要です。
日本語の小説など句読点のぶら下げのスタイルが相応しいものでだけhanging-punctuationを使うべきでしょう。

また、hanging-punctuationを使う場合は、ルート要素に指定するのではなく、段落のスタイルとして指定するのがよいと思います。
日本語の小説などでは段落のスタイルで、`text-indent: 1em` での段落字下げと組み合わせて `hanging-punctuation: first allow-end` を指定するとよいでしょう。
（通常の段落以外にこの hanging-punctuation: first が適用されると、開き括弧類がページ余白の領域にはみ出してしまうので注意が必要です）




```suggestion
  --vs-hanging-punctuation: none;
```
または削除。

---

デフォルトのhanging-punctuationを `none` に変更しました https://github.com/vivliostyle/themes/blob/c70e5fe79f972b34a17d390a22107052ae69eeda/packages/%40vivliostyle/theme-base/css/common/meta-properties.css#L18